package.json

```diff
@@ -84,13 +93,14 @@
     "lasso-less": "^3.0.1",
     "lasso-marko": "^2.4.2",
     "lintspaces-cli": "^0.6.0",
-    "marko": "^3",
+    "marko": "^4",
     "marko-cli": "^3.1.4",
     "marko-dynamic-tag": "^1.0.0",
     "marko-prettyprint": "^1.4.1",
-    "marko-widgets": "^6",
+    "marko-widgets": "^7",
     "mobile-detect": "^1.4.1",
     "mocha": "^5.1.1",
+    "moment": "^2.22.2",
```

Comment:
We get some dependency error if we don't include this. Even upgrading to [v4.0.0-alpha.10 didn't help](https://github.com/storybooks/storybook/issues/3901) (may this will be fixed in v4.0.0). But for now, this seems to be required.